src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -398,4 +411,89 @@ class TableView extends React.Component<TableViewProps> {
   }
 }
 
+const UNSEARCHABLE_FIELDS = ['p99', 'p75', 'p95', 'last_seen'];
```

Comment:
This looks a lot like the list of `AGGREGATE_ALIASES`. Could we use that instead and not have two lists to maintain.

Reply:
I've replaced this with `AGGREGATE_ALIASES`.